qemu/tests/zero_copy.py

```diff
@@ -52,6 +52,9 @@ def enable_zerocopytx_in_host(enable=True):
         enable_zerocopytx_in_host(False)
 
     error.context("Boot vm with 'vhost=on'", logging.info)
+    if params.get("nettype") == "user":
+        raise error.TestNAError("Unable start test with user networking,"
```

Comment:
According to avocado-framework/avocado-vt#941, it is better to use test instance to report the status.